boards/arm64/rpi_5/rpi_5.dts

```diff
@@ -33,6 +35,32 @@
 			label = "ACT";
 		};
 	};
+
+	sd_io_1v8_reg: sd_io_1v8_reg {
```

Comment:
why does the name contain `1v8` if it can be configured to 3.3V?

Reply:
The DT is not invented by me self - it's this way in RPI5 Kernel. I think that, if you wish, you can comment in their project.

Reply:
You requested a review from me, and I asked a question regarding this. I don't see any indication that this code isn't yours. Moreover, you signed off on this code.

If you are not the author and don't understand what is going on here, should you add the original author and license in the header of the file and to the appropriate commit message?

Reply:
i'm not going to comment in this particular thread.